mw-webapp/src/converter/jobDoneConverter.ts

```diff
@@ -0,0 +1,23 @@
+import {querySnapshot} from "src/converter/querySnapshot";
+import {JobDone} from "src/model/businessModel/JobDone";
+import {JobDone as JobDoneDTO} from "src/model/firebaseCollection/JobDone";
+
+export const querySnapshotToJobDoneDTOConverter = (jobsDoneRaw: querySnapshot) => {
```

Comment:
mmm querySnapshot looks too general, do we have any generic for it?

Reply:
Done